rasa/nlu/test.py

```diff
@@ -898,6 +913,100 @@ def compute_metrics(
     return (intent_metrics, entity_metrics, intent_results, entity_results)
 
 
+def compare_nlu(
+    configs: List[Text],
+    data: TrainingData,
+    exclusion_percentages: List[int],
+    f_score_results: Dict[Text, Any],
+    model_names: List[Text],
+    output: Text,
+    runs: int,
+) -> List[int]:
+    """
+    Trains and compares multiple NLU models.
+    For each run and exclusion percentage a model per config file is trained.
+    Thereby, the model is trained only on the current percentage of training data.
+    Afterwards, the model is tested on the complete test data of that run.
+    All results are stored in the provided output directory.
+
+    :param configs: config files needed for training
```

Comment:
We are using the Google syntax for doc strings: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html (you can also change that somewhere in pycharm)